packages/cozy-mjml/README.md

```diff
@@ -0,0 +1,58 @@
+Cozy-MJML
+=========
+
+Cozy-MJML is used for emails. It brings some custom components for MJML that we
+want to share between several projects (cozy-stack, clouderery, notifications,
+etc.).
+
+## How it works?
+
+Cozy-MJML exposes a single binary that behave like mjml CLI. The goal is to
+replace the mjml binary by this one on the cloudery: it will allow to use our
+custom MJML components with mjml-rails, even if the mjml CLI doesn't support
+custom components and mjml-rails has some expectations (like the version number
+of the CLI tools). This binary will also be used by the cozy-stack to generate
+HTML mails: the stack will run it like a konnector or a service.
```

Comment:
Should the email service be an always running server ? I guess the time to generate an email would be then lower than when starting the node process from scratch every time.

Reply:
There are some trade-offs: having an always running server would be faster, but it makes things more complicated to deploy (for self-hosted in particular), monitor, debug. And we don't send that many mails that the performance would be noticeable.